src/nnet3/nnet-example.h

```diff
@@ -50,22 +50,25 @@ struct NnetIo {
   /// the provided features.  t_begin should be the frame that feats.Row(0)
   /// represents.
```

Comment:
You changed the declaration but you didn't change the documentation.   The same below.
And rename to t_stride, as I mentioned for the .cc file.